packages/site/src/data/svelte/blogs.ts

```diff
@@ -17,7 +17,7 @@ export const blogs: Blog<(typeof blogTags)[number]>[] = [
 		title: 'Joy of Code',
 		author: 'Matija',
 		description:
-			"Matija from 🇭🇷 Croatia and he's infinitely curious at how things work but he's mostly passionate about ☕ JavaScript and 🎨 UI/UX design.",
+			"Matija from Croatia and he's infinitely curious at how things work but he's mostly passionate about JavaScript and UI/UX design.",
```

Comment:
Wondering why emoji are being removed. Rendering issues? I know flags sometimes don't render right depending on the OS and browser—had that problem plenty of times on Windows machines. Not sure if that's still the case, or the reason for removing them in this PR.

Reply:
I removed them due to rendering issues. 
They were not rendering correctly on my Mac and I perceive some users might have the same issue. 

I don't think it adds enough value to decide to accommodate it, so I removed it.